test/unit/spec/util/asyncTools.js

```diff
@@ -0,0 +1,54 @@
+define([], () => {
```

Comment:
can you integrate these functions into `test/unit/testUtil.js`?

Reply:
It looks like they were already copied into mswUtils at some point, so I'll extract from here and mswUtils and place into testUtil.js.

Reply:
Actually, waitFor in testUtil.js is not the same thing at all, so best not to incorporate there, or rename it.

Reply:
There is a "wait" in testUtil, but it is based on Bluebird, "waitFor" is plain JS so preferable. 
I think it is best to keep these async utils separate, and to remove the duplicates from mswUtils, as they are unrelated to msw.